README.md

```diff
@@ -96,7 +96,24 @@ When an i18n-static piece is edited, an ID is generated and stored in the `globa
 
 ## 4 Import and export [&#x2B06;](#contents)
 
-The import feature is available in the manager modal: ![](import.jpg)
+It is possible to import and export pieces through existing A3 modules: `npm i @apostrophecms/piece-type-exporter @apostrophecms/piece-type-importer`
```

Comment:
You must also enable those two modules in `app.js`, since we're going into detail.